Source/lighting.cpp

```diff
@@ -806,7 +806,7 @@ void MakeLightTable()
 			}
 		}
 		for (j = 16; j < 20; j++) {
-			col = 8 * j + (shade >> 1);
+			col = 8 * j + (shade / 2);

```

Comment:
Isn't this a real bitshift?

Reply:
Not sure, but you know the code better than me so I'll revert it.